app/controllers/session_controller.rb

```diff
@@ -798,17 +799,20 @@ def failed_to_login(user)
     { error: user.suspended_message, reason: "suspended" }
   end
 
-  def login(user, second_factor_auth_result)
+  def login(user, opts = {})
```

Comment:
```suggestion
  def login(user, passkey_login: false, second_factor_auth_result: nil)
```

I'd explicitly declare the allowed keyword arguments instead of a blanket `opts` argument because it's much less forgiving for typos and less code to type when referencing the arguments (no need for `opts[:...]`)